pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1272,6 +1272,41 @@
+  naipi_page_gaps_learn_more: "Learn more in our [State of CS Report](%{state_of_cs_report_url})."
+  naipi_page_support_heading: "Enhancing access and support"
+  naipi_page_support_desc: "To address the gap in computer science education for Native American students, focused efforts are crucial in enhancing access, particularly in tribal schools. This includes fostering encouragement and support for students, and building strong partnerships among Native American communities, educational institutions, and organizations to promote computer science as a valuable and promising career path."
+  naipi_page_get_involved_heading: "Get involved in expanding access to Computer Science (CS) education"
+  naipi_page_get_involved_educators_title: "Educators"
+  naipi_page_get_involved_educators_desc: "Empower your teaching journey with comprehensive lesson plans, resources, and professional development from Code.org. Gain access to materials tailored for teaching computer science confidently, suitable for educators at all levels of CS teaching experience."
+  naipi_page_get_involved_educators_cta: "Teach with Code.org"
+  naipi_page_get_involved_leaders_title: "School Leaders"
+  naipi_page_get_involved_leaders_desc: "Join us in our mission to create an engaged community focused on Native Excellence. Through a customizable approach, our District Program aims to build community, support increased access to CS education as well as provide districts with unique spotlighting and recognition opportunities."
+  naipi_page_get_involved_leaders_cta: "Discover our District Program"
+  naipi_page_get_involved_students_title: "Students"
+  naipi_page_get_involved_students_desc: "Discover the keys to your next step! Explore opportunities that empower you — from mentorships to scholarships and exciting extracurricular activities. Unleash your potential, find support, and embark on a journey of growth and success in your academic pursuits."
+  naipi_page_get_involved_students_cta: "Discover CS Journeys"
+  naipi_page_get_involved_community_title: "Community Members"
+  naipi_page_get_involved_community_desc: "Partner with us to drive positive change. Explore and share collaboration opportunities, success stories, and the transformative impact we can achieve together in advancing Native American education and computer science. Reach out to us to learn more."
+  naipi_page_get_involved_community_cta: "Email us (naipi@code.org)"
+  naipi_page_engage_heading: "Engage with our community committed to bridging gaps for Native American and Indigenous students"
+  naipi_page_engage_regional_partners: "Regional Partners"
+  naipi_page_engage_other_ways_heading: "Other ways to get involved"
+  naipi_page_engage_other_ways_desc: "Are you a teacher? Indigenous community leader? Funder? Curious about how to get involved? Contact [naipi@code.org](%{email_url}) to explore how we can work together."
+  naipi_page_engage_other_ways_cta_label: "Send us an email"
+
```

Comment:
I really recommend using hierarchy here. Then you should be able to localize strings like this:
`cta_label: hoc_s(:naipi_page.get_involved.educators_cta),`

```suggestion
  naipi_page:
    heading: "Native American and Indigenous Peoples Initiative (NAIPI)"
    intro_heading: "What is NAIPI?"
    intro_desc: "Code.org's Native American and Indigenous Peoples Initiative presents the integration of Native American perspectives and cultural richness in education and technology. Explore innovative approaches shaping Native American education and computer science, discover collaboration opportunities, and witness the collective impact we can achieve together."
    perspectives_heading: "Integrating Native American perspectives"
    perspectives_desc: "Experience a curriculum crafted with care, integrating culturally relevant content that resonates with the rich values and traditions of Native American and Indigenous communities. Elevate your learning journey with resources that truly connect and inspire!"
    module_quilts_aria_primary: "See details for Coding a Geometric Star Quilt module"
    module_quilts_aria_secondary: "Start self-paced professional learning for Coding a Geometric Star Quilt module"
    module_poetry_aria_primary: "See details for Coding with Poetry"
    module_poetry_aria_secondary: "Start self-paced professional learning for Coding with Poetry"
    quote_desc: "Students who are Native American need to have teachers that are sensitive to the tradition and history of the people. I include Native language, examples, and history while I am teaching. At the same time, I recognize that no one person is the same as another and each student needs to have the tools and experiences they need to be able to grow."
    quote_teacher_title: "Library/Media Specialist"
    gaps_heading: "Understanding educational gaps"
    gaps_desc: "Native American communities face unique challenges in offering and engaging in computer science (CS) education in their K-12 systems. Data gathered in the K-12 Computer Science Access Report shows that Native American students are far less likely to attend a school that offers computer science (67% as opposed to the next lowest access group, Black students at 78%)."
    gaps_learn_more: "Learn more in our [State of CS Report](%{state_of_cs_report_url})."
    support_heading: "Enhancing access and support"
    support_desc: "To address the gap in computer science education for Native American students, focused efforts are crucial in enhancing access, particularly in tribal schools. This includes fostering encouragement and support for students, and building strong partnerships among Native American communities, educational institutions, and organizations to promote computer science as a valuable and promising career path."
    get_involved:
      heading: "Get involved in expanding access to Computer Science (CS) education"
      educators_title: "Educators"
      educators_desc: "Empower your teaching journey with comprehensive lesson plans, resources, and professional development from Code.org. Gain access to materials tailored for teaching computer science confidently, suitable for educators at all levels of CS teaching experience."
      educators_cta: "Teach with Code.org"
      eaders_title: "School Leaders"
      leaders_desc: "Join us in our mission to create an engaged community focused on Native Excellence. Through a customizable approach, our District Program aims to build community, support increased access to CS education as well as provide districts with unique spotlighting and recognition opportunities."
      leaders_cta: "Discover our District Program"
      students_title: "Students"
      students_desc: "Discover the keys to your next step! Explore opportunities that empower you — from mentorships to scholarships and exciting extracurricular activities. Unleash your potential, find support, and embark on a journey of growth and success in your academic pursuits."
      students_cta: "Discover CS Journeys"
      community_title: "Community Members"
      community_desc: "Partner with us to drive positive change. Explore and share collaboration opportunities, success stories, and the transformative impact we can achieve together in advancing Native American education and computer science. Reach out to us to learn more."
      community_cta: "Email us (naipi@code.org)"
    engage:
      heading: "Engage with our community committed to bridging gaps for Native American and Indigenous students"
      regional_partners: "Regional Partners"
      other_ways_heading: "Other ways to get involved"
      other_ways_desc: "Are you a teacher? Indigenous community leader? Funder? Curious about how to get involved? Contact [naipi@code.org](%{email_url}) to explore how we can work together."
      other_ways_cta_label: "Send us an email"

```

Reply:
@mgc1194 we don't use this pattern anywhere else on the code.org or hourofcode.com sites that uses this `en.yml` file. Is there a benefit to using this new pattern instead of what we currently do?

Reply:
Hierarchy/namespaces make the strings more readable and discoverable, take a look at [this article](https://lokalise.com/blog/translation-keys-naming-and-organizing/), it was really helpful for me.
We use this hierarchical pattern in [dashboard/config/locales/en.yml](https://github.com/code-dot-org/code-dot-org/blob/d3fc42c8829515374284d29059e89025e3dbdff1/dashboard/config/locales/en.yml#L1409C3-L1418C26)

Reply:
The only two places where we cant use hierarchy is en Apps (javascript does not understand hierarchy) and that i18n spreadsheet (because its a spreadsheet 😆)

Reply:
Additionally, in order to get this PR out (there's fundraising stuff involved with a change I need to make this morning), if we do decide to adopt this new pattern I'd prefer to do it on a different page first ([this one](https://codedotorg.atlassian.net/browse/ACQ-1409)) instead of needing to update all the strings in this PR.

Reply:
It might be worth checking with Jorge but this seems to match the new pattern you were discussing with him, but instead of a prefix (ie `naipe_page_`), it's a hierarchy. As long as it's still easy to group strings by the namespace in Crowdin, I think this makes sense as a pattern to adopt!